src/plugins/Transloadit/index.js

```diff
@@ -120,8 +120,10 @@ module.exports = class Transloadit extends Plugin {
       signature: options.signature
     }).then((assembly) => {
       // Store the list of assemblies related to this upload.
-      const uploadsAssemblies = Object.assign({}, this.state.uploadsAssemblies)
-      uploadsAssemblies[uploadID] = (uploadsAssemblies[uploadID] || []).concat([ assembly.assembly_id ])
+      const assemblyList = this.state.uploadsAssemblies[uploadID]
+      const uploadsAssemblies = Object.assign({}, this.state.uploadsAssemblies, {
+        [uploadID]: assemblyList.concat([ assembly.assembly_id ])
+      })
 
```

Comment:
Should this be `this.state.transloadit.uploadsAssemblies`? If `assemblies` are transloadit-related-only.

Reply:
`this.state` is a getter here that returns `this.core.state.transloadit`. But it's a bit confusing, maybe I should just do `const state = this.core.state.transloadit` at the top of these functions.